### PR TITLE
Add get-predeployed-accounts function

### PR DESCRIPTION
**Pull request by qperrot**
## Usage related changes
- Implementation of get-predeployed-accounts function, which return an array of objects, containing the data of all  pre-deployed accounts (which are: address, initial_balance, private and public keys). 
  - A clientside function for accessing the /predeployed_accounts endpoint on Devnet.

## Development related changes
None

## Checklist:

- [x] I have formatted the code
- [x] I have performed a self-review of the code
- [x] I have rebased to the base branch
- [x] I have documented the changes
- [x] I have updated the tests
- [x] I have created a PR to the `plugin` branch of [`starknet-hardhat-example`]( https://github.com/Shard-Labs/starknet-hardhat-example/pull/66#issue-1278519917).

### CODE REVIEW

File: src/devnet-utils.ts [hunk 1]
```diff
@@ -43,6 +43,13 @@ export interface IncreaseTimeResponse {
     timestamp_increased_by: number;
 }
 
+export interface GetPredeployedAccountsResponse {
```
Comment (FabijanC) [inline src/devnet-utils.ts:46]: You can change the name of this interface to `PredeployedAccount` (singular, not plural)

**Review by FabijanC — CHANGES_REQUESTED** [review]

File: src/devnet-utils.ts (hunk 1 above)
Comment (qperrot) [reply src/devnet-utils.ts]: Fixe bug #eb4a7c4

Comment (qperrot) [reply src/devnet-utils.ts]: Fix bug #125

**Comment by qperrot** [comment]
Fix bug #125

File: src/devnet-utils.ts (hunk 1 above)
Comment (qperrot) [reply src/devnet-utils.ts]: [eb4a7c4](https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/125/commits/eb4a7c43abf7b559bb9760468b4f130d020c3519)

File: scripts/test.sh [hunk 2]
```diff
@@ -9,7 +9,7 @@ CONFIG_FILE_NAME="hardhat.config.ts"
 
 # setup example repo
 rm -rf starknet-hardhat-example
-git clone -b plugin --single-branch git@github.com:Shard-Labs/starknet-hardhat-example.git
+git clone -b predeployed-accounts-test --single-branch git@github.com:Shard-Labs/starknet-hardhat-example.git
```
[the remark this answers is not in the document]
Comment (qperrot) [reply scripts/test.sh]: [9f6fc08](https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/125/commits/9f6fc08d641a5a0cc78ce8910cf54779806f7509)

File: config.json [hunk 3]
```diff
@@ -1,4 +1,9 @@
 {
+<<<<<<< HEAD
```
Comment (FabijanC) [inline config.json:2]: Leftovers from merging.

Comment (quentinShard) [reply config.json]: https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/125/commits/ae2c415563af0673e046190ad9bf1c9a5fb755a1

Comment (quentinShard) [reply config.json]: Update cairo lang
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/125/commits/b379ad98743ca6e7c5d8f4eb443b5b71f961b452

**Pull request merged**